synapse/lib/certdir.py

```diff
@@ -319,19 +319,6 @@ def getHostCaPath(self, name):
 
         return capath
 
-    def getHostCaPath(self, name):
```

Comment:
Possibly submitting a PR from a branch that needs to be rebased to current master?